shared/js/ui/templates/shared/render-broken-site-href.es6.js

```diff
@@ -0,0 +1,6 @@
+
+const bel = require('bel')
+
+module.exports = function (browser, url) {
```

Comment:
I thought this would be helpful as a shared template, since both the hamburger menu and the main popup have this functionality now